bot/exts/info/doc/_cog.py

```diff
@@ -342,9 +342,12 @@ async def get_command(self, ctx: commands.Context, *, symbol_name: Optional[str]
                 error_message = await send_denial(ctx, "No documentation found for the requested symbol.")
                 await wait_for_deletion(error_message, (ctx.author.id,), timeout=NOT_FOUND_DELETE_DELAY)
                 with suppress(discord.NotFound):
-                    await ctx.message.delete()
-                with suppress(discord.NotFound):
-                    await error_message.delete()
+                    await error_message.clear_reaction(Emojis.trashcan)
+
+                if not (ctx.message.mentions or ctx.message.role_mentions):
```

Comment:
Should we perhaps add a comment for this? I feel like it may not be immediately obvious.

Reply:
Are you referring to the if statement?

Reply:
If so, something like `if ...  # if message doesn't ping anyone`?

Reply:
You could add a comment above the if statement (but keeping a new line). For example:
```suggestion
                # Make sure that we won't cause a ghost-ping by deleting the message
                if not (ctx.message.mentions or ctx.message.role_mentions):
```

Reply:
Yeah, that works better. Will do now :+1: